lib/DocumentFilter/DocumentFilterRule.test.js

```diff
@@ -51,52 +54,65 @@ describe('DocumentFilterRule', () => {
     await Button('Submit').exists();
   });
 
-  it('renders Name option', () => {
+  test('renders the correct options for the Attribute dropdown', async () => {
```

Comment:
I separated these deliberately... I want each test to be a single granular check. If this fails we don't know which field broke.